src/ResourceManager/KeyVault/Commands.KeyVault/Commands/AddAzureKeyVaultCertificateContact.cs

```diff
@@ -0,0 +1,115 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.KeyVault.Models;
+using Microsoft.Azure.KeyVault.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+
+
+namespace Microsoft.Azure.Commands.KeyVault
+{
+    /// <summary>
+    /// Adds a given certificate contact to Key Vault for certificate management
+    /// </summary>
+    [Cmdlet(VerbsCommon.Add, CmdletNoun.AzureKeyVaultCertificateContact,
```

Comment:
Again, Add SUpportsShouldProcess-=true and guard execution with a call to ShouldProcess or ConfirmAction